### DISCUSSION TITLE
关于 umi4 的问题

### DISCUSSION THREAD

**Discussion by WuHuSa** [umijs/umi#7916] [General]
umi4 正式版啥时候出，项目从 3 升 4 无痛么

**Comment by chenliangngng** [comment]
现在是rc版，如果新开项目应该没太大问题。我这边老项目升了试试，结果一堆问题就放弃了，等官方文档完善吧

**Comment by sogud** [comment]
3.5还一堆bug没修呢 谁敢用4...
